description/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Description(models.Model):
+    date = models.DateTimeField(default=timezone.now)
+    text = models.TextField()
+    # house_id = models.ForeignKey('user_id', on_delete=models.CASCADE)
```

Comment:
Hi
It will be good to add this function

def __str__(self):
        return self.choice_text

The reason: It’s important to add [__str__()] methods to your models, not only for your own convenience when dealing with the interactive prompt, but also because objects’ representations are used throughout Django’s automatically-generated admin. (https://docs.djangoproject.com/en/4.0/intro/tutorial02/)